frontend/src/components/organizations/OrganizationInfo.vue

```diff
@@ -0,0 +1,230 @@
+                <v-row no-gutters class="ma-2">
+                  <v-col cols="12" md="6" lg="5" xl="4">
+                    <AppLabel>Street address 1:</AppLabel>
+                  </v-col>
+                  <v-col cols="12" md="6" lg="7" xl="8">
+                    {{ organizationInfo?.mailingStreetAddress1 }}
+                  </v-col>
+                </v-row>
+              </v-col>
+              <v-col v-if="organizationInfo?.mailingStreetAddress2" cols="12" md="6">
+                <v-row no-gutters class="ma-2">
+                  <v-col cols="12" md="6" lg="5" xl="4">
+                    <AppLabel>Street address 2:</AppLabel>
+                  </v-col>
+                  <v-col cols="12" md="6" lg="7" xl="8">
+                    {{ organizationInfo?.mailingStreetAddress2 }}
+                  </v-col>
+                </v-row>
+              </v-col>
+            </v-row>
+            <v-row no-gutters>
+              <v-col v-if="organizationInfo?.mailingCity" cols="12" md="6">
+                <v-row no-gutters class="ma-2">
+                  <v-col cols="12" md="6" lg="5" xl="4">
+                    <AppLabel>City:</AppLabel>
+                  </v-col>
+                  <v-col cols="12" md="6" lg="7" xl="8">
+                    {{ organizationInfo?.mailingCity }}
+                  </v-col>
+                </v-row>
+              </v-col>
+              <v-col v-if="organizationInfo?.mailingProvince" cols="12" md="6">
+                <v-row no-gutters class="ma-2">
+                  <v-col cols="12" md="6" lg="5" xl="4">
+                    <AppLabel>Province:</AppLabel>
+                  </v-col>
+                  <v-col cols="12" md="6" lg="7" xl="8">
+                    {{ organizationInfo?.mailingProvince }}
+                  </v-col>
+                </v-row>
+              </v-col>
+              <v-col v-if="organizationInfo?.mailingPostalCode" cols="12" md="6">
+                <v-row no-gutters class="ma-2">
+                  <v-col cols="12" md="6" lg="5" xl="4">
+                    <AppLabel>Postal code:</AppLabel>
+                  </v-col>
+                  <v-col cols="12" md="6" lg="7" xl="8">
+                    {{ organizationInfo?.mailingPostalCode }}
+                  </v-col>
+                </v-row>
+              </v-col>
+            </v-row>
+          </v-col>
+        </v-row>
+      </v-container>
+    </v-skeleton-loader>
+  </v-card>
+</template>
+
+<script>
+import { mapState } from 'pinia'
+import { useAuthStore } from '@/stores/auth'
+import alertMixin from '@/mixins/alertMixin'
+import OrganizationService from '@/services/organizationService'
+import AppLabel from '@/components/ui/AppLabel.vue'
+
+export default {
+  components: { AppLabel },
+  mixins: [alertMixin],
+  data() {
+    return {
+      organizationInfo: undefined,
+      loading: false,
+    }
+  },
+  computed: {
+    ...mapState(useAuthStore, ['userInfo']),
+    isApplicationPage() {
```

Comment:
I'd rather be explicit about allowing edit functionality rather than relying on the path. An allowEdit/editable prop that defaults to false would work well.

Reply:
I added a "editable" prop